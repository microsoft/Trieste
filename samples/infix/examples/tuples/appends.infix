print "empty and empty" append((,), (,));
print "empty append" append(,);
print "unary append" append((,),);
print "nested appends" append((1, 2), append((3,), (4,)));