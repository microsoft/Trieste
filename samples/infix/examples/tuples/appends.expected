//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
//! --enable-tuples calculate
empty and empty <- (,)
empty append <- (,)
unary append <- (,)
nested appends <- (1, 2, 3, 4,)
